### PR TITLE
[Bulk] Validation: Fix various issues in azuredocssdkjava

### PR DESCRIPTION
**Pull request by reemsabaw1**
This PR fixes 100 fixes in azuredocssdkjava including

ms-author-invalid
ms-prod-technology-invalid
docs-link-absolute
hard-coded-locale
ms-prod-and-service
ms-service-subservice-invalid

### CODE REVIEW

File: docs-ref-services/latest/ai-metricsadvisor-readme.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Metrics Advisor client library for Java
 keywords: Azure, java, SDK, API, azure-ai-metricsadvisor, metricsadvisor
+description: Azure Metrics Advisor is a new Cognitive  Service that uses time series based decision AI to identify and assist trouble shooting the incidents of online services, and monitor the business health by automating the slice and dice of business dataFeedMetrics.
```
Comment (Roudabeh1348) [inline docs-ref-services/latest/ai-metricsadvisor-readme.md:4]: remove extra space between Cognitive and Service in line 4

File: docs-ref-services/latest/cognitiveservices/bing-custom-image-search-readme.md [hunk 2]
```diff
@@ -2,10 +2,9 @@
 title: Bing Custom Image Search 
 author: PatrickFarley
 ms.author: pafarley
+description: Bing Search APIs are moving from Cognitive Services to Bing Search Services.
```
Comment (Roudabeh1348) [inline docs-ref-services/latest/cognitiveservices/bing-custom-image-search-readme.md:5]: isn't the description for this one identical to the Azure/azure-docs-sdk-java? but well, if validation is ok with it, then I guess it's fine. :)

Comment (reemsabaw1) [reply docs-ref-services/latest/cognitiveservices/bing-custom-image-search-readme.md:5]: Yeah, that was the only information given in the files so...

**Comment by reemsabaw1** [comment]
@conniey @vcolin7 @johngallardo @g2vinay



The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by samvaity — APPROVED** [review]

**Pull request closed without merging**